provider/marathon/marathon.go

```diff
@@ -369,6 +449,62 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type servicePropertyValues map[string]string
+
+type labelServiceProperties map[string]servicePropertyValues
+
+// Check if for the given application, we find labels that are defining services
+func (p *Provider) hasServices(application marathon.Application) bool {
```

Comment:
Hmm, I see what you mean, but it still seems rather vague to me. I vote for removing the method and ask whoever might need it in the future to file a new issue / PR.